pkgs/os-specific/linux/firmware/rtlwifi_new-firmware/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, lib, linuxPackages }:
+
+with lib;
+
+stdenv.mkDerivation {
+  name = "rtlwifi_new-firmware-${linuxPackages.rtlwifi_new.version}";
+  inherit (linuxPackages.rtlwifi_new) src;
+
+  phases = [ "unpackPhase" "installPhase" ];
```

Comment:
You don't have to do anything if it doesn't have a makefile, ... Otherwise you can do this to disable a specific phase:

```
  buildPhase = ":";
```